.github/workflows/codeql-analysis.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     - name: Setup .NET
       uses: actions/setup-dotnet@v2
       with:
-        dotnet-version: 3.1.300
+        dotnet-version: 6.0.400
```

Comment:
Do you still need to specify the version here? I see it is not specified other yml files

Reply:
Missed it, removed